test/spawning-daemons.js

```diff
@@ -0,0 +1,478 @@
+              expect(node.initialized).to.be.eql(true)
+            })
+
+            it('should be initialized', () => {
+              expect(node.initialized).to.be.eql(true)
+            })
+          })
+        })
+      })
+
+      describe('change config of a disposable node', () => {
+        let ipfsNode
+
+        before((done) => {
+          ipfsd.disposable((err, node) => {
+            if (err) {
+              return done(err)
+            }
+            ipfsNode = node
+            done()
+          })
+        })
+
+        it('Should return a config value', (done) => {
+          ipfsNode.getConfig('Bootstrap', (err, config) => {
+            expect(err).to.not.exist()
+            expect(config).to.exist()
+            done()
+          })
+        })
+
+        it('Should return the whole config', (done) => {
+          ipfsNode.getConfig((err, config) => {
+            expect(err).to.not.exist()
+            expect(config).to.exist()
+            done()
+          })
+        })
+
+        it('Should set a config value', (done) => {
+          async.series([
+            (cb) => ipfsNode.setConfig('Bootstrap', 'null', cb),
+            (cb) => ipfsNode.getConfig('Bootstrap', cb)
+          ], (err, res) => {
+            expect(err).to.not.exist()
+            expect(res[1]).to.be.eql('null')
+            done()
+          })
+        })
+
+        it('should give an error if setting an invalid config value', function (done) {
+          if (isJs) {
+            this.skip()
+          } else {
+            ipfsNode.setConfig('Bootstrap', 'true', (err) => {
+              expect(err.message).to.match(/failed to set config value/)
+              done()
+            })
+          }
+        })
+      })
+
+      it('allows passing via $IPFS_EXEC', (done) => {
+        process.env.IPFS_EXEC = '/some/path'
+        ipfsd.local((err, node) => {
+          expect(err).to.not.exist()
+          expect(node.exec).to.be.eql('/some/path')
+
+          process.env.IPFS_EXEC = ''
+          done()
+        })
+      })
+
+      it('prints the version', (done) => {
+        ipfsd.version((err, version) => {
+          expect(err).to.not.exist()
+          expect(version).to.be.eql(VERSION_STRING)
+          done()
+        })
+      })
+
+      describe('ipfs-api version', () => {
+        let ipfs
+
+        before((done) => {
+          ipfsd.disposable((err, node) => {
+            expect(err).to.not.exist()
+            node.startDaemon((err, ignore) => {
+              expect(err).to.not.exist()
+              ipfs = ipfsApi(node.apiAddr)
+              done()
+            })
+          })
+        })
+
+        // skip on windows for now
+        // https://github.com/ipfs/js-ipfsd-ctl/pull/155#issuecomment-326970190
+        // fails on windows see https://github.com/ipfs/js-ipfs-api/issues/408
+        if (isWindows) {
+          return it.skip('uses the correct ipfs-api')
+        }
```

Comment:
This skip has been removed in https://github.com/ipfs/js-ipfsd-ctl/commit/381ac1c38f0cb7cb19a61b980b1e43844ecb408f, you may have to rebase  https://github.com/ipfs/js-ipfsd-ctl/network